doc/developers/advanced_installation.rst

```diff
@@ -63,7 +63,7 @@ feature, code or documentation improvement).
 
    .. prompt:: bash $
 
-     conda create -n sklearn-env -c conda-forge python=3.9 numpy scipy cython
+     conda env create -f environments/common.yml
```

Comment:
I strongly dislike this since one cannot say what the name of the environment should be.

Reply:
One may use `conda env create -f environments/common.yml -n <env_name>` -- I added a note below this section to address this.

Reply:
nice, didn't know that.